src/pages/unified-preload.js

```diff
@@ -128,6 +128,19 @@ try {
     // Settings pages get full electronAPI access (exactly what settings.js expects)
     contextBridge.exposeInMainWorld('electronAPI', {
       settings: settingsAPI,
+      getTabs: () => ipcRenderer.invoke('get-tabs'),
+      closeTab: (id) => ipcRenderer.invoke('close-tab', id),
+      activateTab: (id) => ipcRenderer.invoke('activate-tab', id),
+      groupAction: (action, groupId) => ipcRenderer.invoke('group-action', { action, groupId }),
+      updateGroupProperties: (groupId, properties) => ipcRenderer.send('update-group-properties', groupId, properties),
+      onGroupPropertiesUpdated: (callback) => {
+        ipcRenderer.on('group-properties-updated', (_, groupId, properties) => {
+          callback(groupId, properties);
+        });
+      },
+      onVerticalTabsChanged: (callback) => createEventListener('vertical-tabs-changed', callback),
```

Comment:
The toggle is not working. Check the recordings below.

https://github.com/user-attachments/assets/88925f48-a517-4a85-b584-ef07710eb3ec

https://github.com/user-attachments/assets/b8884688-90b4-45b4-8dd5-503427ea9c05

Reply:
It's not updating the URL

https://github.com/user-attachments/assets/2073030b-e530-4b32-9c61-5d5cf0facbbc